Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Iterative_authalic_parameterizer_3.h

```diff
@@ -632,41 +631,33 @@ class Iterative_authalic_parameterizer_3
   /// \param mesh a triangulated surface.
   /// \param main_vertex_v_i the vertex of `mesh` with index `i`
   /// \param neighbor_vertex_v_j the vertex of `mesh` with index `j`
-  NT compute_w_ij(const Triangle_mesh& tmesh,
+  NT compute_w_ij(const Triangle_mesh& mesh,
                   vertex_descriptor main_vertex_v_i,
                   Vertex_around_target_circulator<Triangle_mesh> neighbor_vertex_v_j) const
   {
-    const PPM ppmap = get(vertex_point, tmesh);
-
-    const PPM_ref position_v_i = get(ppmap, main_vertex_v_i);
-    const PPM_ref position_v_j = get(ppmap, *neighbor_vertex_v_j);
+    const PPM ppmap = get(vertex_point, mesh);
+    const Point_3& position_v_i = get(ppmap, main_vertex_v_i);
```

Comment:
Why change away from PPM_ref?

Reply:
I took this code from other parameterizers because the authors originally also seemed to copy it from there. But you are right, `PPM_ref` is better in this context. I fixed that.